assets/js/util.js

```diff
@@ -3,7 +3,7 @@
 
 var SPACE_KEY_CODE = 32, ENTER_KEY_CODE = 13,
     HTTP_OK_CODE = 200, HTTP_BAD_REQUEST_CODE = 400,
-    XHR_LOADING = 3, XHR_DONE = 4;
+    XHR_LOADING = 3, XHR_DONE = 4, SUGGESTION_DESTROY_TIMEOUT = 3000;
```

Comment:
Put it in translator.js then :) Also, this a bit too technical of a name for a var that can be edited by a user.